server/src/test/java/org/elasticsearch/cluster/coordination/PublicationTests.java

```diff
@@ -178,6 +178,7 @@ public void testSimpleClusterStatePublishing() throws InterruptedException {
             discoveryNodes, singleNodeConfig, singleNodeConfig, 42L), ackListener, Collections.emptySet());
 
         assertThat(publication.pendingPublications.keySet(), equalTo(discoNodes));
+        assertThat(publication.completedNodes().size(), equalTo(0));
```

Comment:
I prefer using the `empty()` matcher so if it's not empty we get to see what was in it.